server/src/graphql/resolvers/mutations.js

```diff
@@ -340,6 +340,18 @@ module.exports = {
     }
   },
 
+  upsertInfrastructureRequirement: async (_, { model }) => {
+    if (!model.id) {
+      model.id = uuidv4();
```

Comment:
Do you need to do this since you set a default value on `id`? If it isn't provided, the table will generate a uuid. Or at least it should. If it doesn't, then y'all should probably just stop using the default value

Reply:
Oh. I see that you need to create the id here because you are using upsert which only returns a boolean.